### PR TITLE
Add docs for integration testing in Front

### PR DESCRIPTION
**Pull request by eps1lon**
This has been available for a while but not documented.

### CODE REVIEW

**Comment by eps1lon** [comment]
* **#69172** <a href="https://app.graphite.dev/github/pr/vercel/next.js/69172?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @eps1lon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: contributing/core/testing.md [hunk 1]
```diff
@@ -157,3 +157,22 @@ or, to use `next` from a specific Pull Request (PR number required):
...
+
+### Integration testing on Front
+
+To test a preview build on Front, you can use the `next` package from the preview build URL.
+It is recommended to use the dedicated GitHub Workflow for that which also takes care of syncing versions across the whole repo, updating React etc:
+
+```bash
+# while the current working directory is vercel/front
+# Uses a build from the PR 67571
+$ gh workflow run cron-update-next.yml --fiel version=#67571
```
Comment (mischnic) [inline contributing/core/testing.md:169]: Should probably be this?
```suggestion
$ gh workflow run cron-update-next.yml --field version=#67571
```

File: contributing/core/testing.md [hunk 2]
```diff
@@ -157,3 +157,22 @@ or, to use `next` from a specific Pull Request (PR number required):
...
+# Uses a build from commit 188f76947389a27e9bcff8ebf9079433679256a7
+$ gh workflow run cron-update-next.yml --raw-field version=@188f76947389a27e9bcff8ebf9079433679256a7
+
+# or, from any current working directory
+# Uses a build from the PR 67571
+$ gh workflow run --repo vercel/front cron-update-next.yml --fiel version=#67571
```
Comment (eps1lon) [inline contributing/core/testing.md:175]:
```suggestion
$ gh workflow run --repo vercel/front cron-update-next.yml --field version=#67571
```

File: contributing/core/testing.md (hunk 1 above)
Comment (eps1lon) [reply contributing/core/testing.md]: thanks for catching!

**Review by mischnic — APPROVED** [review]

**Comment by eps1lon** [comment]
Closed in favor of internal docs

**Pull request closed without merging**